geemap/common.py

```diff
@@ -16158,3 +16158,118 @@ def is_on_aws():
         if item.endswith(".aws") or "ec2-user" in item:
             on_aws = True
     return on_aws
+
+
+def google_map_tiles(
```

Comment:
I thought about this approach as well. One potential issue is that different map types will accept different optional parameters, such as `layerTypes` and `style`. It would not work well if users want to use different optional parameters for different map types, like this:
```python
google_map_tiles(region="CN", language="zh-Cn", layerTypes=["layerTraffic"], style=style)
```